lib/bootloader_setup.pm

```diff
@@ -482,30 +482,34 @@ sub uefi_bootmenu_params {
 
     # Kiwi in TW uses grub2-mkconfig instead of the custom kiwi config
     # Locate gfxpayload parameter and update it
-    if (is_jeos && (!is_sle('=12-SP5') || is_opensuse)) {
+    # The main branch should be used only for bootable pre-installed images that contain already full
+    # grub2 configuration
+    if (get_var('BOOT_HDD_IMAGE') && (is_jeos || is_leap_micro || is_microos || is_sle_micro)) {
```

Comment:
is this applicable to older versions or only from 6.1 onwards?

Reply:
This should work on all versions. At least that is the intention